### PR TITLE
Init gitignore

### PR DESCRIPTION
**Pull request by ghost**
Feel free to edit this if you feel there's any files here that we should keep updated.

### CODE REVIEW

File: .gitignore
Comment (gareth-d-ga) [inline .gitignore:13]: Here and below, how is the `**` notation different from `*`

File: .gitignore
Comment (gareth-d-ga) [inline .gitignore:20]: Just double checking that this only ignores binary files without an extension? In other contexts the notation would suggest that all files match.

File: .gitignore
Comment (ghost) [reply .gitignore:13]: From the documentation:
An asterisk "*" matches anything except a slash.
A slash followed by two consecutive asterisks then a slash matches zero or more directories.

I wasn't sure if there were any additional folders in the examples that may create these files, so I put the double asterisks to catch it. But it's quite possible that it's redundant.

**Review by gareth-d-ga — COMMENTED** [review]
Thanks Matt,

As mentioned yesterday, the way I use git, I don't find problems with untracked files. So ATM I'm not well placed to evaluate this.

Might be easiest to discuss later today.

File: .gitignore
Comment (ghost) [reply .gitignore:20]: Yes, together lines 17-19 ignores any file without an extension inside the propagation/SWALS/examples directory.

File: .gitignore
Comment (gareth-d-ga) [reply .gitignore:13]: Ahh, so this version would also match  `propagation/SWALS/examples/OUTPUTS`, whereas a single `*` would not?

Comment (ghost) [reply .gitignore:13]: Yes I believe so, and it would also match `propagation/SWALS/examples/my_folder/OUTPUTS` as well as  `propagation/SWALS/examples/my_folder/another_folder/OUTPUTS`

**Comment by gareth-d-ga** [comment]
Following our chat today, I'm happy to merge this after you add those couple of comments to the file. Thanks!

**Comment by ghost** [comment]
I added in those comment in the gitignore file. But I'll still leave it up to you to merge.

**Review by gareth-d-ga — APPROVED** [review]

**Pull request merged**
